Dockerfile

```diff
@@ -20,14 +20,11 @@ COPY requirements.txt requirements.txt
 RUN pip3 install --no-cache-dir -r requirements.txt
```

Comment:
Can you please merge in mainline - these changes should already be corrected

Reply:
And please revert changes

Reply:
How does this work? Can I merge without a PR approval?